src/global/helpers.js

```diff
@@ -119,6 +129,7 @@ module.exports = {
 	isForced,
```

Comment:
Die ganzen helper sind in meinen PR schon raus gehauen. Daher nur das drin was wir wirklich benötigen.

Reply:
Ok willst du das dann hier kurz machen bzw. mit dem merge, ich weis gerade nicht was davon noch relevant ist.